src/components/common/PaginatedTxns/index.tsx

```diff
@@ -43,6 +43,20 @@ const TxPage = ({
 
   const isQueue = useTxns === useTxQueue
 
+  useEffect(() => {
+    if (!isQueue) {
+      return
+    }
+
+    // If conflict transactions split across pages, we must fetch the next page
+    const lastItem = page?.results[page.results.length - 1]
+    const hasSplitConflict = lastItem && isHasNextConflictTransactionListItem(lastItem)
```

Comment:
What happens if we don't fetch the next page in those cases and what does the list look like when paginating then?

Reply:
They don't appear as a full group.